src/components/podcasts_list/PodcastList.js

```diff
@@ -22,7 +23,7 @@ export const PodCastComponent = ({track, onClick}) => (
                 <div className="float-end" id="download">
                     <a href={track.fileUrl} className="border-light bg-transparent text-dark"
                        type="download">
-                        <i className="fa-solid fa-down-to-bracket"  aria-hidden="true"/>

```

Comment:
Oh! @godsakani please some changes need to be done here

Reply:
Hello @yokwejuste 
ok  should undo it?

Reply:
Let's discuss first on that please.

Reply:
> Let's discuss first on that please.

ok sir 
so what is the issue now?